docs/dxp/latest/en/liferay-internals/extending-liferay/overriding-module-language-keys.md

```diff
@@ -1,6 +1,10 @@
-# Overriding Module Language Keys
+# Overriding Module Language Keys in Earlier Versions
 
-The language keys of a module can be overridden just like how [global language keys can be overridden](./overriding-global-language-keys.md). The steps are slightly different when working with a module. 
+```{important}
+If you're working with Liferay DXP/Portal 7.4+, please follow the instructions for [Overriding Global Language Keys](./overriding-global-language-keys.md).
+```
+
+Overriding Liferay application specific language keys in earlier versions is similar to overriding global language keys in earlier versions but there are additional steps. 
```

Comment:
Creating E6U7EnglishResourceBundle in the example below is the same as the new extension point, it just has more steps and code. So I think there's no required migration process.

If they have multiple language override bundles they can easily update them with the new `liferay.language.resources` header and remove the E6U7EnglishResourceBundle component/class and "liferay.resource.bundle" bnd capabilities.